utils/test/oo_config_tests.py

```diff
@@ -3,12 +3,14 @@
 # pylint: disable=bad-continuation,missing-docstring,no-self-use,invalid-name
 
 import os
-import unittest
-import tempfile
 import shutil
+import tempfile
+import unittest
+
 import yaml
+from ooinstall.models import InvalidHostError
+from ooinstall.oo_config import Host, OOConfig
```

Comment:
I'm not sure why this doesn't raise errors when I run CI on it. `ooinstall.oo_config` shouldn't have `Host` in it anymore. I'd think this should be more like this:

> from ooinstall.models import InvalidHostError, Host
> from ooinstall.oo_config import OOConfig